### DISCUSSION TITLE
config still says version 7.2

### DISCUSSION THREAD

**Discussion by J4k-The-Reaper** [UltimaHoarder/UltimaScraper#1374] [General]
i might be asking a dumb question here.....but why does config/.settings still say version 7.2 at top? shouldn't it say 7.4, since the latest update? no complaints here...just honestly curious. plus, it still has the starsvn on bottom. that is supposed to be removed correct??

**Comment by UltimaHoarder** [comment]
The config is it's own version. Whenever I make a change, I increase it to the script's latest version.

**Reply by J4k-The-Reaper** [reply]
ahhhh....that makes sense....told ya i asked dumb question....lol

**Reply by J4k-The-Reaper** [reply]
last thing, how do i use the "ability to skip downloading content"? sorry...i am a total noob at this stuff....trying to learn by others.

**Reply by UltimaHoarder** [reply]
Set downloader to false in the config

**Reply by J4k-The-Reaper** [reply]
awesome...guess i coulda saw that for myself.....what exactly does this new feature do?

**Reply by marinefl74** [reply]
> awesome...guess i coulda saw that for myself.....what exactly does this new feature do?

Heya @J4k-The-Reaper hopefully this isnt breaking protocol but I had a couple questions for you re. our conversation last week when I was having issues with the auth.json. All is working great, my question(s) are specific to what you mentioned about Git and OneDrive. Is there a way to PM you or email you so I'm not cluttering the board? 

Again, apologies, very new to git so I'm learning as I go. Thanks!
